core/kazoo_number_manager/src/knm_number.erl

```diff
@@ -528,10 +533,10 @@ is_force_outbound(PN) ->
 
 -spec is_force_outbound(ne_binary(), ne_binary(), boolean()) -> boolean().
 is_force_outbound(?NUMBER_STATE_PORT_IN, Module, _ForceOutbound) ->
-    kapps_config:get_is_true(?KNM_CONFIG_CAT, <<"force_port_in_outbound">>, true)
+    knm_config:should_force_port_in_outbound()
         orelse force_module_outbound(Module);
 is_force_outbound(?NUMBER_STATE_PORT_OUT, Module, _ForceOutbound) ->
-    kapps_config:get_is_true(?KNM_CONFIG_CAT, <<"force_port_out_outbound">>, true)
+    knm_config:should_force_port_out_outbound()
```

Comment:
The use of an app-specific config module is debatable to me: what does it bring?
Everywhere in Kazoo calls to kapps_config are inside macros defined at the top of a module.